integration_tests/gdk/components/test_integ_BuildCommand.py

```diff
@@ -148,7 +147,6 @@ def test_build_run_default_zip_json(mocker, supported_build_system, rglob_build_
         mock_json_dump.call_count == 1
 
     assert mock_get_proj_config.assert_called_once
-    assert not mock_subprocess_run.called
```

Comment:
Nit: LGTM or we can assert that maven/gradle build commands are not called using subprocess like this

`assert not mock_subprocess.called_with_args(['mvn','clean','package'])`

Reply:
But that's a growing list too! Whenever we there is a new build strategy added, this test would need a change, so I refrained from using that.